### ISSUE TITLE
Доработать переход из Карты счета на документ по товару

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2769]
Originally reported on Google Code with ID 2769

```

    Например, формируем Карту счета по 01 счету. Здесь имеет место быть аналитика "Товар".
Задав в параметрах Карты счета конкретный товар или отфильтровав данные при нажатии
на кнопку "Изменить документ" переход осуществляется на документ с отображением всех
позиций.
    Можно ли доработать переход из Карты счета на документ по товару с отображением
ТОЛЬКО определенного товара ? Это удобно, когда в документе много товаров.

```

Reported by `NikolayUkleyko` on 2012-03-30 15:20:15

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-03-30 15:37:07
- **Labels added**: Type-Enhancement, Priority-Medium, GedeminEXE, Depot
- **Labels removed**: Gomeloboi, Etalon

**Comment by gsbelarus** [comment]
```

 Так можно ли это сделать или нет ?

```

Reported by `NikolayUkleyko` on 2013-05-17 12:07:59

**Comment by gsbelarus** [comment]
```
Вообще так нельзя делать (я имею ввиду открытие документа не со всем списком). Можно
еще попробовать сделать позиционирование на нужную позицию - это наверное можно сделать,
но придется дорабатывать окно складского документа. А так у нас есть кнопка которая
открывает окно с конкретной позицией, но к сожалению окно позиции складского документа
почти ни где не настроено так как не имеет практической ценности... 
```

Reported by `mikleshoihet` on 2013-05-17 12:28:04

**Comment by gsbelarus** [comment]
```

  Хотя бы сделайте автоматическое позиционирование на нужную позицию (ТМЦ).

```

Reported by `NikolayUkleyko` on 2013-05-18 07:55:31

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2014-08-06 11:32:21
- **Status changed**: `Duplicate`
- **Merged into**: #3403
